website/src/index-page.tsx

```diff
@@ -1,119 +1,374 @@
+          <FeatureHighlights
+            highlights={[
+              {
+                title: 'Passes all audits',
+                description: (
+                  <>
+                    Tested using the graphql-http library.&nbsp;
+                    <TextLink
+                      href="https://github.com/enisdenjo/graphql-http/blob/master/implementations/graphql-yoga/README.md"
+                      className="text-primary-500"
+                    >
+                      See the report
+                    </TextLink>
+                    &nbsp;for more info.
+                  </>
+                ),
+                icon: (
+                  <Image
+                    src={httpImage}
+                    loading="eager"
+                    placeholder="empty"
+                    alt="Server over HTTP"
+                    className="w-full h-full"
+                  />
+                ),
+              },
+              {
+                title: 'Subscriptions',
+                description: (
+                  <>
+                    Built-in GraphQL Subscriptions over&nbsp;
+                    <TextLink
+                      href="https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events"
+                      className="text-primary-500"
+                    >
+                      Server-Sent Events
+                    </TextLink>
+                    .
+                  </>
+                ),
+                icon: (
+                  <Image
+                    src={subscriptionsImage}
+                    loading="eager"
+                    placeholder="empty"
+                    alt="Subscriptions"
+                    className="w-full h-full"
+                  />
+                ),
+              },
+              {
+                title: 'File uploads',
+                description: (
+                  <>
+                    Through GraphQL out-of-the box leveraging the&nbsp;
+                    <TextLink
+                      href="https://github.com/jaydenseric/graphql-multipart-request-spec"
+                      className="text-primary-500"
+                    >
+                      GraphQL multipart request specification
+                    </TextLink>
+                    .
+                  </>
+                ),
+                icon: (
+                  <Image
+                    src={uploadsImage}
+                    loading="eager"
+                    placeholder="empty"
+                    alt="Uploads"
+                    className="w-full h-full"
+                  />
+                ),
+              },
+            ]}
+          />
+        </div>
+      </Feature>
+      <Feature
+        title="Extend the GraphQL Engine"
+        description={
+          <>
+            powered by&nbsp;
+            <TextLink
+              href="https://www.the-guild.dev/graphql/envelop"
+              className="text-primary-500"
+            >
+              envelop
```

Comment:
```suggestion
              Envelop
```
Envelop should be always capitalized

Reply:
Sure, I thought the brand name is "envelop".

Reply:
I remember Envelop uses capitalized version, cc @n1ru4l

Reply:
Just checked, you're right @B2o5T.